src/EmulatorEntry.cpp

```diff
@@ -89,6 +89,7 @@ void EmulatorEntry::start(int argc, char **argv) {
         }
     }
 
+    std::filesystem::path databaseFilePath = std::filesystem::current_path() / ".." / ".." / "dependencies" / "chip-8-database" / "database";
```

Comment:
What is the purpose of this? It does not seem to be used? Plus there is the one in the `Constants.h` file